app/suite/file.go

```diff
@@ -107,102 +94,89 @@ func (a *App) TestFileStoreConnectionWithConfig(cfg *model.FileSettings) *model.
+func (a *SuiteService) FileSize(path string) (int64, *model.AppError) {
+	size, nErr := a.platform.FileBackend().FileSize(path)
 	if nErr != nil {
 		return 0, model.NewAppError("FileSize", "api.file.file_size.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 	return size, nil
 }
 
-func (a *App) FileModTime(path string) (time.Time, *model.AppError) {
-	modTime, nErr := a.FileBackend().FileModTime(path)
+func (a *SuiteService) FileModTime(path string) (time.Time, *model.AppError) {
+	modTime, nErr := a.platform.FileBackend().FileModTime(path)
 	if nErr != nil {
 		return time.Time{}, model.NewAppError("FileModTime", "api.file.file_mod_time.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 
 	return modTime, nil
 }
 
-func (a *App) MoveFile(oldPath, newPath string) *model.AppError {
-	nErr := a.FileBackend().MoveFile(oldPath, newPath)
+func (a *SuiteService) MoveFile(oldPath, newPath string) *model.AppError {
+	nErr := a.platform.FileBackend().MoveFile(oldPath, newPath)
 	if nErr != nil {
 		return model.NewAppError("MoveFile", "api.file.move_file.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 	return nil
 }
 
-func (a *App) WriteFile(fr io.Reader, path string) (int64, *model.AppError) {
-	return a.Srv().writeFile(fr, path)
-}
-
-func (s *Server) writeFile(fr io.Reader, path string) (int64, *model.AppError) {
-	result, nErr := s.FileBackend().WriteFile(fr, path)
+func (a *SuiteService) WriteFile(fr io.Reader, path string) (int64, *model.AppError) {
+	result, nErr := a.platform.FileBackend().WriteFile(fr, path)
 	if nErr != nil {
 		return result, model.NewAppError("WriteFile", "api.file.write_file.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 	return result, nil
 }
 
-func (a *App) AppendFile(fr io.Reader, path string) (int64, *model.AppError) {
-	result, nErr := a.FileBackend().AppendFile(fr, path)
+func (a *SuiteService) AppendFile(fr io.Reader, path string) (int64, *model.AppError) {
+	result, nErr := a.platform.FileBackend().AppendFile(fr, path)
 	if nErr != nil {
 		return result, model.NewAppError("AppendFile", "api.file.append_file.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 	return result, nil
 }
 
-func (a *App) RemoveFile(path string) *model.AppError {
-	return a.Srv().removeFile(path)
-}
-
-func (s *Server) removeFile(path string) *model.AppError {
-	nErr := s.FileBackend().RemoveFile(path)
+func (a *SuiteService) RemoveFile(path string) *model.AppError {
+	nErr := a.platform.FileBackend().RemoveFile(path)
 	if nErr != nil {
 		return model.NewAppError("RemoveFile", "api.file.remove_file.app_error", nil, "", http.StatusInternalServerError).Wrap(nErr)
 	}
 	return nil
 }
 
-func (a *App) ListDirectory(path string) ([]string, *model.AppError) {
-	return a.Srv().listDirectory(path, false)
+func (a *SuiteService) ListDirectory(path string) ([]string, *model.AppError) {
+	return a.listDirectory(path, false)
```

Comment:
Same as above.

Reply:
What about cases like this one? This one doesn't do any error mapping

Reply:
Oh, this is different. These were `app.Server` methods and `app.App` was wrapping them, since I moved them all and merged under `SuiteService` now they look weird. I can update those for sure.